VocaDbWeb/Program.cs

```diff
@@ -17,6 +18,17 @@
 
 	startup.ConfigureServices(builder.Services);
 
+	builder.Services.AddRateLimiter(options =>
+	{
+		options.RejectionStatusCode = StatusCodes.Status429TooManyRequests;
+		options
+			.AddSlidingWindowLimiter(policyName: "details", options =>

```

Comment:
`"details"` should probably be a `public const string` in a constants class instead of a magic value that needs to be specified in multiple places.